SPECS/krb5/CVE-2024-37370.patch

```diff
@@ -26,8 +26,8 @@ version_fixed: 1.21.3
  src/lib/gssapi/krb5/k5sealv3.c    |   5 +
  src/lib/gssapi/krb5/k5sealv3iov.c |   3 +-
  src/lib/gssapi/krb5/k5unsealiov.c |  80 +++++++++-
- src/tests/gssapi/t_invalid.c      | 233 +++++++++++++++++++++++++-----
- 4 files changed, 275 insertions(+), 46 deletions(-)
```

Comment:
Why did you remove the tests? They seem to be present in the [original CVE patch](https://github.com/krb5/krb5/commit/55fbf435edbe2e92dd8101669b1ce7144bc96fef#diff-39f0036d8943271a8f7510ab2dafef5a3a4b8f35d36a3395a65447d9bf1c630b).

Reply:
Replied in email thread as well but adding here: The new tests that are added as part of the patch cause the t_invalid.c tests to fail as there is an abort() being called on one of the return values of an internal function. I These specific tests work fine on 1.21.3 but given that they are having issues in 1.19.4, I believe there has to be some changes in between the versions in their codebase that is impacting the behavior of some of these internal functions.